doc/faq.rst

```diff
@@ -87,3 +89,27 @@ You can build pyopencl against conda's OpenCL driver in the following way::
    # Apply this patch on MacOS: https://raw.githubusercontent.com/conda-forge/pyopencl-feedstock/master/recipe/osx_flags.diff
    $ ./configure.py --cl-inc-dir=$PWD/../miniforge3/envs/ceesd/include --cl-lib-dir=$PWD/../miniforge3/envs/ceesd/lib
    $ pip install -e .
+
+How can I record the exact versions of Python packages that are currently installed?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Running emirge's `version.sh` script creates a requirements.txt file that
+stores the exact git commits of each emirge sub-repository that were used at
+the time when `version.sh` was executed. You can use this file to install the
+exact versions of the packages at a later time::
+
+   $ cd emirge/
+   $ ./version.sh --output-requirements=myreq.txt
+   [...]
+   *** Creating requirements file with current emirge module versions
+   [...]
+   *** Created file 'myreq.txt'. Install it with 'pip install --src . -r myreq.txt'.
+
+   $ pip install --src . -r myreq.txt
```

Comment:
Also mention that the same `requirements.txt` can be used to create a new environment. Looks good otherwise.

Reply:
added.